src/msgs/history2.sql

```diff
@@ -1144,7 +1144,7 @@ message to appear.', 'Your program encountered an input or output error.', 'c_pg
 (1149, 'hvlad', '2006-02-02 19:15:11', 17, 44, 'Current memory = !c
 Delta memory = !d
 Max memory = !x
-Elapsed time= !e sec', NULL, NULL, 'c_pg');
+Elapsed time = !e sec', NULL, NULL, 'c_pg');
 (1150, 'hvlad', '2006-02-03 19:17:10', 17, 45, 'Cpu = !u sec
```

Comment:
history2.sql contains old versions of changed messages. Formally, you should put pre-edited version of message as new record there, and never change existing records. But i really not sure it (history2.sql) is used somehow these days...